core/witness_generator/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
```

Comment:
What is this? I don't see any files in this project.

Reply:
sorry, garbage.